### DISCUSSION TITLE
Plot3D and Name File Conversion to CGNS format?

### DISCUSSION THREAD

**Discussion by ChrisO56** [CGNS/CGNS#316] [CGNStalk]
I have a .p3d filr with a .nam for boundaries. Is there away to convert both the .p3d file to CGNS while preserving the named boundary conditions from the .nam file?

**Comment by dbgarlisch** [comment]
Have you tried...
https://cgns.github.io/CGNS_docs_current/cgnstools/utilities/import.html

**Comment by ChrisO56** [comment]
I took a look at the utilities for plot3d to CGNS and the q file seems to be only the initialization/solution, not boundary condition naming. Unless I am misunderstanding the options for that part of the utilities.
